### PR TITLE
Networking and information changes

### PR DESCRIPTION
**Pull request by kforeverisback**
…etween network impl

# Purpose of PR
- Added a link to diff of various network impl
- Flannel CIDR change NOTE
- 
## Type of PR

- [X] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Documentation updated

### CODE REVIEW

**Review by AAkindele — APPROVED** [review]

File: Lab.md [hunk 1]
```diff
@@ -19,17 +19,18 @@
...
+  -g $AKDC_RG \
+  --admin-username akdc \
+  -n akdc \
+  --size standard_d2s_v3 \
+  --nsg-rule SSH \
+  --image Canonical:UbuntuServer:18.04-LTS:latest \
+  --os-disk-size-gb 128 \
+  --generate-ssh-keys \
+  --query publicIpAddress -o tsv \
+  --custom-data scripts/startup.sh)
```
Comment (AAkindele) [inline Lab.md:32]: Heads up. The user is asked to change to the scripts directory here. https://github.com/retaildevcrews/akdc#clone-this-repo
You can add a `cd` command to the top of this section. Another option is to move this note to the top of this section and be specific about being in the scripts directory. That way, the user reads the notice first, before running the command.

File: Lab.md [hunk 2]
```diff
@@ -216,7 +237,12 @@ kubectl config use-context kubernetes-admin@kubernetes
 
 Kubernetes uses a plugin architecture for the network. It has requirements for it's network. Network plugins then provide the implementation details that satisfy those requirements. A couple of options are Flannel and Calico.
 
+<<<<<<< HEAD
+For more information on other avaiable network implementations, go here. [Network implementations](https://kubernetes.io/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model).
+Also check out this [table of comparison](https://biarca.io/2018/10/choosing-the-right-network-for-your-kubernetes-cluster/) between different network implementation.
+=======
 For more information about other available network implementations, go here. [Network implementations](https://kubernetes.io/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model).
+>>>>>>> origin/main
```
Comment (AAkindele) [inline Lab.md:245]: There's a merge conflict here.

```suggestion
For more information about other available network implementations, go here. [Network implementations](https://kubernetes.io/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model).
Also check out this [table of comparison](https://biarca.io/2018/10/choosing-the-right-network-for-your-kubernetes-cluster/) between different network implementation.
```

**Review by AAkindele — CHANGES_REQUESTED** [review]

File: Lab.md (hunk 1 above)
Comment (kforeverisback) [reply Lab.md]: Oh I thought this Lab.md was standalone.
Let me add a `cd akdc/scripts` before this cmd.

**Review by kforeverisback — COMMENTED** [review]
Changes done

**Review by AAkindele — APPROVED** [review]

**Pull request merged**